MC/config/PWGGAJE/external/generator/generator_pythia8_powheg.C

```diff
@@ -0,0 +1,102 @@
+R__ADD_INCLUDE_PATH($O2DPG_MC_CONFIG_ROOT)
+///#include "FairGenerator.h"
+//#include "Generators/GeneratorPythia8.h"
+#include "Pythia8/Pythia.h"
+#include "Generators/GeneratorPythia8Param.h"
+// Pythia8 generator with POWHEG
+//
+// Author: Marco Giacalone (marco.giacalone@cern.ch)
+
+// o2-sim-dpl-eventgen --nEvents 10 --generator external\
+   --configKeyValues "GeneratorExternal.fileName=${O2DPG_MC_CONFIG_ROOT}/MC/config/PWGGAJE/external/\
+   generator/generator_pythia8_powheg.C;GeneratorExternal.funcName=\
+   getGeneratorJEPythia8POWHEG(\"powheg.input\",\"${O2DPG_MC_CONFIG_ROOT}/MC/config/PWGGAJE/pythia8/generator/pythia8_powheg.cfg\")"
+// or with iniFile
+// o2-sim -g external --noGeant -n 2 -j 8 --configFile $O2DPG_MC_CONFIG_ROOT/MC/config/PWGGAJE/ini/GeneratorPythia8POWHEG.ini
+
+namespace o2
+{
+namespace eventgen
+{
+
+using namespace Pythia8;
+
+class GeneratorJEPythia8POWHEG : public o2::eventgen::GeneratorPythia8
```

Comment:
I think it would be useful to make a short description/comment of what this generator class does.